tests/go/GoReturnsBearTest.py

```diff
@@ -29,6 +29,5 @@
 
 GoReturnsBearTest = verify_local_bear(
```

Comment:
Also correct GoReturnsBear's run() documentation to show using goreturns instead of using gofmt, as sils suggested.